public/assets/css/bloop.css

```diff
@@ -16,6 +16,8 @@
 .team-lead-card img {
   width: 100%;
   border-radius: 10px 10px 0px 0px;
+  height: 240px;
```

Comment:
We can probably get rid of this as currently the site will crop them to a 1:1 aspect ratio by itself and this can cause it to not be 1:1 at certain resolutions. If this is causing any issues with photos you can manually crop the team lead photos to a square aspect ratio